schedule_lessons/dashboard/static/dashboard/js/settings.js

```diff
@@ -0,0 +1,11 @@
+$(document).ready(function () {
```

Comment:
can we use ES6 style arrow functions here?
```
$(document).ready(() => {

});
```

Reply:
For sure!